### PR TITLE
day 10 & 11 beta test and misc edits

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 1/01_00_ARC01_2.txt [hunk 1]
```diff
@@ -702,7 +702,7 @@ It happen again? You collapse from anemia this morning or something?"
 -- Page 122, Offset 1473.
 -- 心配してくれるのはありがたいんだけどね、そう四六時中貧血を起こしてたら体がもたないよ」
 It's nice of you to worry about me,
- but if I were anemic 24/7 my body would have withered away by now."
+ but if I were anemic 24/7, my body would have withered away by now."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 1/01_00_ARC01_2.txt:705]: This 2 comma construction basically always strikes me as 'textbook' and overly slow for short spoken sentences like this

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 2]
```diff
@@ -31,7 +31,7 @@ As I bask in the morning's warm welcome―――
 [sha:6cbbd7b766f75ada376a8dfb0bfea9c54ddc9de8]{
 -- Page 2, Offset 24265.
 -- 無意識のうちに枕元の眼鏡を手に取る。
-I put on the glasses resting on my nightstand,
+I put on my glasses resting on the nightstand,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:34]: This construction feels slightly off to me, as though he could be the one resting on the nightstand. Either revert or maybe adjust to something like
> I pick up my glasses from the nightstand and put them on, my hands...

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 3]
```diff
@@ -94,7 +94,7 @@ My train of thought is totally derailed.
 [sha:bc196e99dfd03b131a1d1f51c80d79a4876fd8e6]{
 -- Page 6, Offset 24277.
 -- 脳が豆腐になったかのような真っ白さ。
-My mind is so blank it's as though my brain has turned into tofu.
+My mind is so blank, it's as though my brain has turned into tofu.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:97]: Don't love this comma

File: script/Arcueid/Day 10/10_00_ARC10_3B.txt [hunk 4]
```diff
@@ -196,7 +196,8 @@ What's with that casual wear? That fur? That ribbon?
 [sha:a7fe8a94b8b046f93a8ed87d31090cad1bebb4ff]{
 -- Page 110, Offset 24537.
 -- 言うに事欠いて、おま、おまお、おまえ、おまえーーーー！
-There was no need to ask that! Yo―You, you, you―――!
+// why a dash not a hyphen here?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3B.txt:199]: I think we've generally used ascii `-` for stutters?

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 5]
```diff
@@ -142,7 +142,7 @@ It makes total sense that Arcueid would turn heads just by walking around the pa
 [sha:a63c77499f4c105887af1d9c79e415b5fb2a431e]{
 -- Page 252, Offset 24901.
 -- そればかりか、アルクェイドに声をかけるタイミングを窺っている男たちもいる。
-Not only that, but there also seem to be a few men waiting for the right time to make a pass at her.
+Not only that, but there also seems to be a few men waiting for the right time to make a pass at her.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:145]: revert

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 6]
```diff
@@ -164,7 +164,7 @@ Unlike the plaza around the fountain, our normal rendezvous spot is devoid of pe
 -- Page 255, Offset 24905.
 -- 結局ここか、とガッカリしないでもないが、先ほどよりは幾分ましだ。
 I'd be lying if I said I wasn't a little disappointed that this is where we ended up,
- but at least it beats where we were before.
+ but at least, it beats where we were before.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:167]: objectionable comma

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 7]
```diff
@@ -241,12 +241,12 @@ As lunchtime arrives, the number of businesspeople in suits multiplies.
 [sha:e240d0ae0eabeec11cc0beedee4c9177f1c2c91d]{
 -- Page 261, Offset 24920.
 -- 俺と同年代の学生たちも増えてきた。
-The tally of students in my age range goes up as well.
+The tally of students in my age group goes up as well.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:244]: revert

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 8]
```diff
@@ -241,12 +241,12 @@ As lunchtime arrives, the number of businesspeople in suits multiplies.
...
 }
 [sha:d330392691aaf3341e16c5773ed2cea59e2526f6]{
 -- Page 261, Offset 24921.
 -- それと、母親に連れられて無邪気に走りまわっている子供たち。
-There seem to be more children around too,
+There seems to be more children around too,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:249]: revert, children are plural

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 9]
```diff
@@ -424,7 +424,7 @@ From earlier? So it's not about my weird line of questioning?
 [sha:d6e023ab630ac1640670e65b1366fbe046cb39b6]{
 -- Page 274, Offset 24956.
 -- 志貴には何度か触れられてるけど、それは理由があっての事でしょう？　けどさっきのは理由がなかったから、何だったのかなって」
-You've touched me a number of times before, but each time you did you had a reason, right?
+You've touched me a number of times before, but each time you did, you had a reason, right?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:427]: feels like too many commas

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 10]
```diff
@@ -28,7 +28,7 @@ But that devilish smile of hers steamrolled any resistance I might have mustered
 [sha:67c23261b35b3ecbac5ab24de49135b67249d959]{
 -- Page 398, Offset 25268.
 -- 午後５時前。
-Five o'clock.
+It's now five o'clock.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:31]: Not sure I like `now`, it implies some progression that isn't really here. Maybe just `It's five o'clock` or `It's five in the evening`.

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 11]
```diff
@@ -13,7 +13,7 @@ The loud crack of an explosion splits the air in front of me,
 [sha:e676279c8f6e9f4e7ed2be8b025f1c54ecb48c4f]{
 -- Page 626, Offset 25937.
 -- 「うっわあ、恐かったあ……！　いちかばちかでちょっかいだしたけど、反撃されなくて助かったぁ……！」
-"Uwah, that was scary...! I just took a chance and went for it, but thank God she didn't fight back...!"
+"%{i}Uwah%{/i}, that was scary...! I just took a chance and went for it, but thank God, she didn't fight back...!"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:16]: nyet comma

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 12]
```diff
@@ -189,12 +189,13 @@ Fighting monsters' supposed to be you people's fuckin' job, ain't it?"
 [sha:7c168ecf191decd5ee00a6da17fa1e080c37d878]{
 -- Page 636, Offset 25971. Mods: @k, @e.
 -- 「あわわ、あわわわわ！　ああもう、後でぜぇっっったい、
+// is that italics at promise fine?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:192]: yeah it's emphasis that's ok

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 13]
```diff
@@ -461,6 +461,7 @@ It seems like splitting hairs, but he's not wrong.
 // Reading 人間以上 as more of a 'superhuman', above/beyond human capabilities rather than _ranking_
 // since yeah otherwise this feels like a flip
 // In the last sentence he's talking about their immortality. here is talking about their overall capabilities.
+// why is this worded like Mario is talking?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:464]: Is this Shiki? I thought this was still Mario, and the うん in the next line is the start of Shiki's reply

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 14]
```diff
@@ -559,7 +560,7 @@ They call these Hemonomic Principles―――or Idea Blood.
 [sha:1ffcd9cecab50231034cd2935ac635b51f9c014c]{
 -- Page 708, Offset 26177.
 -- で、だ。そんな死徒たちの中に、真剣に不老不死を学んでいやがるクソ野郎がいやがった」
-Alright, so: among those Dead Apostles, there was one motherfucker who really fuckin' went at it studyin' immortality..."
+Alright, so among those Dead Apostles, there was one motherfucker who really fuckin' went at it studyin' immortality..."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:563]: revert

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 15]
```diff
@@ -1256,6 +1257,7 @@ His manner suggests that he no longer has any interest in me or my fate.
 [sha:986d20bee85ed5ac1ea6e756829372ed0eef9cd4]{
 -- Page 747, Offset 26304.
 -- 貴方も貴方よ!?　さっき噛まれかけたクセに、まだあの怪物の本性が分からないの!?」
+// should -kun be added here since that's how Noel calls Shiki?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:1260]: She doesn't actually say his name here, we reified it. I guess we could add the suffix?

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 16]
```diff
@@ -1332,7 +1334,7 @@ Where exactly do you intend to go from here?"
 [sha:5dc556a226d5c40dc5f8d6787b70995b42c51dab]{
 -- Page 752, Offset 26318.
 -- ……生か死かの選択を迫る声。
-Her voice presses me to choose between life and death...
+Her voice presses me to choose between life or death...
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:1337]: revert

File: script/Arcueid/Day 10/10_00_ARC10_7.txt [hunk 17]
```diff
@@ -18,7 +18,7 @@ Also you ain't got the skills to take down this kid.
 -- Page 756, Offset 26324.
 -- 「なんでって、そろそろ<調律|ピアノ>の影響が出る頃だからだよ。
 // His gloves are the 'Piano Machine'
-"Well, 'cause the effects of the Piano's 'boutta kick in any minute, that's why.
+"Well, 'cause the effects of the %{i}Piano's%{/i} 'boutta kick in any minute, that's why.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_7.txt:21]: looks a bit weird to be with the `'s` in there, might just not italics this one and let the capitals indicate it's Special

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt [hunk 18]
```diff
@@ -530,6 +530,7 @@ The first thing that I feel is pure survival instinct.
 -- Page 802, Offset 26486.
 -- ここにいてはいけない。アレを見てはいけない。
 // This is very minor, but "mustn't" isn't really used in American English. This could be changed to "shouldn't" if it matters.
+// should we follow that comment above?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_7_2B.txt:533]: I personally think must is more urgent but I'm also a filthy en_UK speaker

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt [hunk 19]
```diff
@@ -596,7 +597,7 @@ First comes fear.
 [sha:4e284ae945f7197ec263ff869b3fe4fc9e251019]{
 -- Page 804, Offset 26499.
 -- その次に来たものは、歓喜に似た殺意だった。
-But after it follows an almost rapturous urge to kill.
+But after that, follows an almost rapturous urge to kill.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_7_2B.txt:600]: revert

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 20]
```diff
@@ -965,7 +965,7 @@ The only time she was allowed to leave was when she had been assigned a new targ
 [sha:722ebbe0f00a1203607b30e5dd09650b58a42ff4]{
 -- Page 100, Offset 29784.
 -- その時に応じた知識を教えこまれて、確実に敵を処刑できるように」
-When that happened, she would be taught only as much as was necessary in order for her to execute her mark."
+When that happened, she would be taught only as much as necessary in order for her to execute her mark."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:968]: might revert

File: script/Arcueid/Day 11/11_00_ARC11_3.txt [hunk 21]
```diff
@@ -1482,7 +1483,7 @@ Is her wound hurting...?
 -- いろんな感情が溢れてきて、体がぽわっとして、頬が自分のものじゃないみたい。自分がいまどんな顔をしているか分からないくらいに。
 // ぽわっと is very likely ほわっと but with a だくてん for emphasis
 I'm overflowing with emotions right now,
- my body is so flooded with relief it's like I've lost control over my cheeks.
+ my body is so flooded with relief, it's like I've lost control over my cheeks.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_3.txt:1486]: prefer without comma

File: script/Arcueid/Day 1/01_00_ARC01_2.txt (hunk 1 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 1/01_00_ARC01_2.txt]: yeah that's why i thought it was correct :necoarc:
ill refrain from adding those in voiced lines if it doesn't work

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt (hunk 18 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 10/10_00_ARC10_7_2B.txt:533]: :soubrit:
let's wait for the american's opinion then

File: script/Common/00_02.txt [hunk 22]
```diff
@@ -146,6 +146,7 @@ My chest is pounding so hard that it feels like I can barely breathe.
 [sha:30abf38b77fd3a31a43f7c785091becbb376609c]{
 -- Page 6, Offset 34.
 -- ざあ　と<梢|こずえ>の窓が開いて、
+// why is one in italics?
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Common/00_02.txt:149]: i think it looks better with but consistency on something like this isn't that important.

**Review by Avuxo — APPROVED** [review]
otsukaresama
nothing is ever hyphenated except when its not hyphenated and then its hyphenated :sacchincopium:

**Pull request merged**